### ISSUE TITLE
pureperms prevetion

### ISSUE THREAD

**Issue by nikoskon2003** [TesseractTeam/Tesseract#3]
if you have pure perms installed on the server you cannot join it!

**Comment by daniktheboss** [comment]
Plugin issue

**Comment by ObeyBeastJj23** [comment]
Its not a plugin issue (https://github.com/PurePlugins/PurePerms/issues/268)

But on the other hand, install an older PurePerms and it will work fine for you 😃 
Hope this works for me and you together as this is a must have for servers.

    - Obey

**Comment by AppleDevelops** [comment]
Yeah, we’d have to update our stuff to work with PMMP, but I don’t have plans to make Tesseract the same as PMMP

**Comment by AppleDevelops** [comment]
I'll go ahead and leave this open until I make some sort of compatibility with PurePerms

**Comment by nikoskon2003** [comment]
@AppleDevelops ok nice!
hope it is fixed soon :D

**Comment by AppleDevelops** [comment]
test now

**Comment by nikoskon2003** [comment]
@AppleDevelops works perfectly
